src/test/groovy/de/tracetronic/jenkins/plugins/ecutestexecution/steps/GenerateReportsStepIT.groovy

```diff
@@ -78,6 +78,12 @@ class GenerateReportsStepIT extends IntegrationTestBase {
             st.assertRoundTrip(step, 'ttGenerateReports additionalSettings: [' +
                     "[name: 'javascript', value: 'False']], failOnError: false, generatorName: 'HTML', " +
                     "reportIds: ['1', '3']")
+        when:
+            step.setReportIds("1, ,3")
```

Comment:
better update to something different than the values from before. Currently, if setReportIds would do nothing, the test would still pass